mtp_noms_ops/apps/security/urls.py

```diff
@@ -375,7 +375,7 @@ def dashboard_view(request):
         name='resolve_check',
     ),
     url(
-        r'^security/checks/(?P<check_id>\d+)/assignment/$',
+        r'^security/checks/(?P<check_id>\d+)/assignment/(?P<list>\d+|)$',
```

Comment:
i think a word would be better than an integer. you have to search around to figure out what options exist. is it just 2 urls that we need in the end? maybe you could just have "/security/checks/???/assignment/" and "/security/checks/???/assignment/list/"

Reply:
so maybe `r'^security/checks/(?P<check_id>\d+)/assignment/(?P<list>list|)/$'` (including the trailing slash!)